src/utils/shared.py

```diff
@@ -309,11 +309,18 @@ async def remove_download(
         if removeFromClient:
             logger.info(">>> Removing %s download: %s", failType, affectedItem["title"])
         else:
-            logger.info(
-                ">>> Removing %s download (without removing from torrent client): %s",
-                failType,
-                affectedItem["title"],
-            )
+            if settingsDict['SET_OBSOLETE_QBIT_TAG'] and settingsDict['OBSOLETE_QBIT_TAG']:
+                logger.info(
+                    ">>> Removing %s download (without removing from torrent client but with setting obsolete tag): %s",
```

Comment:
">>> Removing %s download (keeping it in torrent client and adding tag '%s'

where %s is settingsDict['OBSOLETE_QBIT_TAG'] 


proposal above.